src/frontend/screens/Library/components/InstallModal/ModDialog/index.tsx

```diff
@@ -0,0 +1,343 @@
+            </div>
+            <div className={styles.stepSubtitle}>
+              {t(
+                'mods.instructions.step1.subtitle',
+                'Download World of Warcraft 3.3.5a'
+              )}
+            </div>
+            <span className={styles.stepDescription}>
+              {t('mods.instructions.step1.description', {
+                defaultValue:
+                  'This game is a mod for World of Warcraft: Wrath of the Lich King (3.3.5a). In order to play this mod, players must own a copy of World of Warcraft 3.3.5a. {{newline}} {{newline}} HyperPlay does not host World of Warcraft. Players who own World of Warcraft can obtain version 3.3.5a from several external sources:',
+                newline: '\n'
+              })}
+            </span>
+            <ul className={styles.sourceList}>
+              {/*               <li className={styles.sourceItem}>
+                <a
+                  className={styles.sourceLink}
+                  href={downloadLinks[appName]?.webtorrent ?? ''}
+                  onClick={(ev) => {
+                    window.api.openExternalUrl(
+                      downloadLinks[appName]?.webtorrent
+                    )
+                    ev.preventDefault()
+                  }}
+                >
+                  {t(
+                    'mods.instructions.step1.webTorrent',
+                    'WebTorrent (recommended)'
+                  )}
+                </a>
+              </li> */}
+              {/*               <li className={styles.sourceItem}>
+                <div className={styles.magnetContainer}>
+                  <a
+                    href={downloadLinks[appName]?.magnet ?? ''}
+                    className={styles.sourceLink}
+                    onClick={(ev) => {
+                      window.api.openExternalUrl(
+                        downloadLinks[appName]?.magnet ?? ''
+                      )
+                      ev.preventDefault()
+                    }}
+                  >
+                    {t(
+                      'mods.instructions.step1.bitTorrent',
+                      'BitTorrent Magnet Link'
+                    )}
+                  </a>
+                  <ButtonCopy
+                    text={downloadLinks[appName]?.magnet ?? ''}
+                    className={styles.copyMagnetButton}
+                  />
+                </div>
+              </li> */}
+              <li className={styles.sourceItem}>
+                <a
+                  href={downloadLinks[appName]?.chromieCraft ?? ''}
+                  className={styles.sourceLink}
+                  onClick={(ev) => {
+                    window.api.openExternalUrl(
+                      downloadLinks[appName]?.chromieCraft ?? ''
+                    )
+                    ev.preventDefault()
+                  }}
+                >
+                  {t('mods.instructions.step1.chromieCraft', 'ChromieCraft')}
+                </a>
+              </li>
```

Comment:
When I click chromiecraft, it opens file explorer to the folder I am running HyperPlay from

Reply:
damn, changed the logic and forgot to test it again.
Fixed it now.